content/guides/understanding-inbox-messaging.md

```diff
@@ -0,0 +1,77 @@
+# Understanding Inbox Messaging
+
+---
+
+1. TOC
+{:toc}
+
+---
+
+## Preface
+
+BookingSync provides the ability to integrate with the Inbox on multiple of the different channels/platforms we support. This guide will present
+how to implement this integration.
+
+
+## Purpose
+
+The purpose of this feature is to provide a seamless integration between internal messaging systems in certain Channels and the BookingSync platform, in order for adminstrators to be able to handle incoming inquiries in a centralized location.
+
+
+## Entities
+
+- **Inbox Conversation**: Represents a thread/conversation and is scoped under a Channel, an Account, and a Booking.
+<br>
+- **Inbox Participant**: Represents a participant in the thread and is scoped under an Inbox Conversation. There are two types of participants
+  - Client: Represents a Traveller, managed by the Channel entirely.
+  - Host: Represents a User in the Inbox Conversation's Account, managed by the Channel and the Account. All Users should be able to view the Inbox Conversations of their corresponding Account. However, Only Host Inbox Participants could send messages and be notified about new messages in the Inbox Conversation.
+<br>
+- **Inbox Message**: Represents a message in the thread/conversation and is scoped and an Inbox Participant.
+
+
+## Flow
+
+When a Booking is created on the following channels: (Booking.com, Airbnb), a corresponding Inbox Conversation is created on our system, which is synchronised with the Channel internal messaging systems.
+
+When an Inbox Conversation is created via a Channel, at least one Client Inbox Participant will be added to the conversation automatically. A Host Inbox Participant may or may no be created via the Channel automatically.
+
+
+## Fetching an Inbox Conversation
+
+There are multiple ways to fetch an existing Inbox Conversation.
+
+1. Using the [Inbox Conversations endpoints](/reference/endpoints/inbox_conversations), you could get all the Inbox Conversations related to your account.
+2. Using the [Bookings endpoints](/reference/endpoints/application) `conversations` field, which you could then query the [Inbox Conversations endpoints](/reference/endpoints/inbox_conversations) with the provided ids to get Inbox Conversations related to a certain Booking.
+
+In order to discern which Channel this Inbox Conversation is associated with, you can check `created_by_id` and `created_by_type` attributes of the Inbox Conversation. `created_by_type` should be `Doorkeeper::Application` and `created_by_id` should refer to the Application id. Applications can be identified in [Applications endpoint](/reference/endpoints/applications).
```

Comment:
I'm not sure we have to mention `Doorkeeper` here as we don't have any mentions of what `Doorkeeper::Application` is in the whole api docs repo. So maybe just omit the "Doorkeeper" part? What do you think?

Reply:
We shouldn't have a mention to it because as far as this guide is concerned, it is just a constant value. This is just a string value that does not carry any context.